### PR TITLE
Remove test dependecies on specific relay clients

### PR DESCRIPTION
**Pull request by serban300**
Related to https://github.com/paritytech/parity-bridges-common/issues/2538

Removing some dependencies on specific relay clients from generic code

### CODE REVIEW

**Review by svyatonik — APPROVED** [review]

File: relays/lib-substrate-relay/Cargo.toml [hunk 1]
```diff
@@ -58,6 +58,4 @@ sp-runtime = { git = "https://github.com/paritytech/polkadot-sdk", branch = "mas
 [dev-dependencies]
 bp-rococo = { path = "../../chains/chain-rococo" }
```
Comment (bkontur) [inline relays/lib-substrate-relay/Cargo.toml:59]: @serban300 can we also remove this? `lib-substrate-relay` should be generic, right?

Comment (serban300) [reply relays/lib-substrate-relay/Cargo.toml:59]: Oops, you're right. Thanks ! I'll open a follow-up PR.

Comment (serban300) [reply relays/lib-substrate-relay/Cargo.toml:59]: Done: https://github.com/paritytech/parity-bridges-common/pull/2900 . PTAL

Comment (bkontur) [reply relays/lib-substrate-relay/Cargo.toml:59]: nice, thank you :)

**Pull request merged**